Duckling/Time/EN/Rules.hs

```diff
@@ -1118,6 +1118,30 @@ ruleIntervalFromDDDDMonth = Rule
       _ -> Nothing
   }
 
+ruleIntervalFromDDDDOfMonth :: Rule
+ruleIntervalFromDDDDOfMonth = Rule
+  { name = "from the <day-of-month> (ordinal or number) to the <day-of-month> (ordinal or number) of <named-month> (interval)"
+  , pattern =
+    [ regex "from( the)?"
+    , Predicate isDOMValue
+    , regex "\\-|to( the)?|th?ru|through|(un)?til(l)?"
+    , Predicate isDOMValue
+    , regex "of"
```

Comment:
For some reason, I am unable to get this to work for "of".

Reply:
It's because when pattern matching `tokens` in the production rule, you're missing the new regex match token.